tests/autoyast/installation.pm

```diff
@@ -173,15 +175,15 @@ sub run {
     $checktime = 30;
     $looptime  = 0;
     $timeout   = 0;
-    while (!$timeout) {
+    while ( !$timeout ) {
 
         mouse_hide(1);
-        $ret = check_screen(["reboot-after-installation", "autoyast-error"], $checktime);
+        $ret = check_screen( [ "reboot-after-installation", "autoyast-error" ], $checktime );
 
         #repeat until timeout or login screen
-        if (defined $ret) {
-            if (match_has_tag('autoyast-error')) {
-                record_soft_failure 'AUTOYAST_EXPECT_ERRORS ' . get_var('AUTOYAST_EXPECT_ERRORS_REASON');
+        if ( defined $ret ) {
+            if ( match_has_tag('autoyast-error') ) {
+                diag( 'Expected error. Iteration = ' . $i );
```

Comment:
record_info would be better, nobody will know about "diag" until he reads log
I remember I added AUTOYAST_EXPECT_ERRORS_REASON when I was workarounding some bugs, so I would leave also the record_soft_failure as possible workaround in case of some bug e.g. record soft failure if AUTOYAST_EXPECT_ERRORS and AUTOYAST_EXPECT_ERRORS_REASON is defined ?